packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -39,10 +38,14 @@ const localBuilders: { [key: string]: BuilderWithPackage } = {
   },
 };
 
-const bundledBuilders = Object.keys(devDependencies).filter(d =>
+const bundledBuilders = Object.keys(pkg.devDependencies).filter(d =>
   d.startsWith('@now/')
 );
 
+export function getBundledBuilders() {
+  return ['@now/go', '@now/next', '@now/node'];
```

Comment:
What about `@now/python` and `@now/ruby`?

Reply:
I'm not sure, they weren't part of the package.json before, but I've added them

cc @TooTallNate